content/en/security_platform/application_security/getting_started/go.md

```diff
@@ -0,0 +1,96 @@
+---
+title: Go Applications
+kind: documentation
+code_lang: go
+type: multi-code-lang
+code_lang_weight: 20
+further_reading:
+    - link: 'https://github.com/DataDog/dd-trace-go/tree/v1'
+      tag: 'GitHub'
+      text: 'Go Datadog Library source code'
+---
+
+{{% appsec-getstarted %}}
+- [CGO][1] and a C toolchain are enabled in your build environment.
+
+## Get started
+
+1. **Add or update your program's dependencies** with the latest version of the Datadog Go library, at least version 1.36.0:
+   ```
+   go get -v -u gopkg.in/DataDog/dd-trace-go.v1@v1.36.0
+   ```
+
+2. **Recompile your program by adding the `appsec` build tag** using the Go compiler option `-tags appsec`:
+   ```
+   go build -tags appsec my-program
+   ```
+
+3. **Redeploy your Go service and enable Application Security** by setting the `DD_APPSEC_ENABLED` environment variable to `true`:
+   ```
+   env DD_APPSEC_ENABLED=true ./my-program
+   ```
+   Or one of the following methods, depending on where your application runs:
+
+   {{< tabs >}}
+{{% tab "Docker CLI" %}}
+
+Update your configuration container for APM by adding the following argument in your `docker run` command: 
+
+```
+docker run [...] -e DD_APPSEC_ENABLED=true [...] 
+```
+
+{{% /tab %}}
+{{% tab "Dockerfile" %}}
+
+Add the following environment variable value to your container Dockerfile:
+
+```
+ENV DD_APPSEC_ENABLED=true
+```
+
+{{% /tab %}}
+{{% tab "Kubernetes" %}}
+
+Update your configuration yaml file container for APM and add the AppSec env variable:
+
+```
+spec:
+  template:
+    spec:
+      containers:
+        - name: <CONTAINER_NAME>
+          image: <CONTAINER_IMAGE>/<TAG>
+          env:
+            - name: DD_APPSEC_ENABLED
+              value: "true"
+```
+
+{{% /tab %}}
+{{% tab "AWS ECS" %}}
+
+Update your ECS task definition JSON file, by adding this in the  environment section:
+
+```
+"environment": [
+  ...,
+  {
+    "name": "DD_APPSEC_ENABLED",
+    "value": "true"
+  }
+]
+```
+
+{{% /tab %}}
+
+{{< /tabs >}}
+
+{{% appsec-getstarted-2 %}}
+
+{{< img src="/security_platform/application_security/application-security-signal.png" alt="Security Signal details page showing tags, metrics, suggested next steps, and attacker IP addresses associated with a threat." style="width:100%;" >}}
+
+## Further Reading
```

Comment:
Advanced installation details (extracted from our Sqreen documentation https://docs.sqreen.com/go/installation/#advanced-installation):

```suggestion
## Requirements installation

The compilation of AppSec involves [CGO][1] and therefore requires:

- The `gcc` compiler for the target `GOOS` and `GOARCH`.
- The C library headers.
- The CGO bindings enabled. This is controlled by the `CGO_ENABLED` environment
  variable which is enabled by default when compiling natively.  
  If CGO is disabled, major agent features such as RASP and WAF will be
  disabled.

Examples of requirements installation on common systems:

{{< tabs >}}

{{% tab "Debian/Ubuntu" %}}
    ```console
    $ apt install gcc libc6-dev
    ```
{{% /tab %}}

{{% tab "Alpine" %}}
    ```console
    $ apk add gcc musl-dev
    ```
{{% /tab %}}

{{% tab "RHEL/CentOS/Fedora" %}}
    ```console
    $ yum install gcc glibc-devel
    ```
{{% /tab %}}

{{% tab "macOS" %}}
    ```console
    $ xcode-select --install 
    ```
{{% /tab %}}

{{< /tabs >}}

Please note that the Go toolchain disables CGO when cross-compiling.

## Further Reading
```

Reply:
I have added this content, but put it on the Setup and Config page where the rest of that kind of information is going. It's linked to from the prereq section on this Go Getting Started page.

Reply:
Perfect, thank you very much.